content/en/getting_started/synthetics/browser_test.md

```diff
@@ -39,7 +39,11 @@ The example below demonstrates the creation of a browser test that maps a user's
 
 1. In the Datadog site, hover over **Digital Experience** in the left hand menu and select **[Tests][2]** (under **Synthetic Monitoring & Testing**).
 2. In the top right corner, click **New Test** > **[Browser Test][3]**.
-3. Define your browser test:
+3. Select a pre-populated template to choose from, or build a test from scratch.
+
+   {{< img src="getting_started/synthetics/synthetics_browser_templates.png" alt="Synthetics browser test landing page with templates" style="width:100%;" >}}
+
+4. Define your browser test (build a test from scratch):
```

Comment:
Steps 3 and 4 here confused me a bit. In step 3, "or build a test from scratch" is an option, but then step 4 is building a step from scratch as well? Would it be possible to clarify the steps here to make it easier to figure out where the options branch off from each other?

Reply:
Hi @janine-c this is now ready to go, including mobile tests, I re-worked most of the pages slightly so it probably needs another full review if you have time? Thanks!

Reply:
Thanks for the heads up, @aliciascott ! I'm on it!